Source/Core/DolphinWX/ARCodeAddEdit.cpp

```diff
@@ -30,7 +30,7 @@ CARCodeAddEdit::CARCodeAddEdit(int _selection, std::vector<ActionReplay::ARCode>
 	Bind(wxEVT_BUTTON, &CARCodeAddEdit::SaveCheatData, this, wxID_OK);
 
 	ActionReplay::ARCode tempEntries;
-	wxString currentName = _("Insert name here...");
+	wxString currentName = _("");
```

Comment:
You don't need the `= _("")`. What you're saying is this should be translated, which is not needed for an empty string.